modules/041-linstor/templates/nodegroupconfiguration-disable-lvmetad.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: deckhouse.io/v1alpha1
+kind: NodeGroupConfiguration
+metadata:
+  name: disable-lvmetad.sh
+  {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
+spec:
+  weight: 101
+  nodeGroups: ["*"]
+  bundles: ["*"]
+  content: |
+    # Copyright 2023 Flant JSC
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    #     http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+
+    # The lvmetad service should not be used by LVM on the node.
+    # So we disable the service and set the setting use_lvmetad to 0 in lvm.conf.
+
+    configure_lvm_disable_lvmetad() {
+      command -V lvmconfig >/dev/null 2>&1 || return 0
```

Comment:
1. Don't use `||` trick, it is difficult to read.
2. Is it possible if there isn't a lvmconfig command?